src/themes/uswitch/theme.json

```diff
@@ -700,6 +700,7 @@
 
     "a": {
       "color": "inherit",
+      "fontWeight": 500,
```

Comment:
Approved, but should this not be added to fontWeights?

Reply:
Yeah I was thinking that but couldn't come up with a name for it 🤔 . Happy to change later on